lib/ts/recipe/dashboard/api/userroles/permissions/getPermissionsForRole.ts

```diff
@@ -0,0 +1,36 @@
+import UserRoles from "../../../../userroles";
+import { APIInterface, APIOptions } from "../../../types";
+
+import STError from "../../../../../error";
+
+const getPermissionsForRole = async (
+    _: APIInterface,
+    ___: string,
+    options: APIOptions,
+    __: any
+): Promise<{
+    status: "OK";
+    permissions: string[];
+}> => {
+    const role = options.req.getKeyValueFromQuery("role");
+
+    if (role === undefined || typeof role !== "string") {
+        throw new STError({
+            message: "Required parameter 'role' is missing or has an invalid type",
+            type: STError.BAD_INPUT_ERROR,
+        });
+    }
+
+    const response = await UserRoles.getPermissionsForRole(role);
+
+    if (response.status === "UNKNOWN_ROLE_ERROR") {
+        return {
+            status: "OK",
+            permissions: [],
+        };
+    }
```

Comment:
why are we doing this? We should return UNKNOWN_ROLE_ERROR to the frontend, and the frontend should handle this in the proper way.. we need to distinguish between role not existing vs role existing with no permissions?